scripts/benchmarks/parse-benchmarks-data.js

```diff
@@ -78,12 +65,29 @@ if (performanceYear >= 2020) {
   MCC_BENCHMARK_CSV_COLUMNS.push('isHighPriority');
 }
 
-// New 2022 data update
-if (performanceYear >= 2022) {
+// New 2023 data update
+if (performanceYear >= 2023) {
   BENCHMARK_CSV_COLUMNS.push('isInverse');
```

Comment:
@peg-uehling Chris and I made the decision to remove `isInverse` and `metricType` from the expected csv columns for 2022 data only. When the script expects those columns for 2022, it breaks when running the current csv containing 2022 data. If we end up loading new 2022 data with these fields, we will get a helpful developer error, and we can simply add-back those columns here.

Reply:
It these fields are supposed to be option for 2022 according to this ticket:
https://jira.cms.gov/browse/QPPA-6971